bedrock/mozorg/templates/mozorg/internet-health/open-innovation.html

```diff
@@ -0,0 +1,412 @@
+              {% trans cc='https://stateof.creativecommons.org/2015/' %}
+                Source: <a href="{{ cc }}">Creative Commons</a>, 2015
+              {% endtrans %}
+              </cite>
+            </p>
+          </div>
+          {# L10n: this is used as a decorative element in a callout box. #}
+          <span class="aside-accent" aria-hidden="true" role="presentation">{{ _('1B') }}</span>
+        </aside>
+      </div>
+    </section>
+
+    <section class="topic-action">
+      <div class="content">
+        <div class="topic-action-you">
+          <h3 class="topic-action-title">{{ _('How you can engage in copyright reform') }}</h3>
+          <ul>
+            <li>
+              <h4>{{ _('Keep the pressure on') }}</h4>
+              <p>
+              {% trans single_market='https://ec.europa.eu/commission/priorities/digital-single-market_en',
+                       change='https://www.changecopyright.org/' %}
+                Protests against SOPA/PIPA in the US and ACTA in Europe led to the
+                shelving of laws that may have hindered open innovation. EU citizens: the
+                <a href="{{ single_market }}">Digital Single Market</a> could produce
+                good or bad new copyright law. <a href="{{ change }}">Join us</a> and
+                others in the fight!
+              {% endtrans %}
+              </p>
+            </li>
+            <li>
+              <h4>{{ _('Get creative') }}</h4>
+              <p>
+              {% trans attribution='https://wiki.creativecommons.org/wiki/Best_practices_for_attribution' %}
+                Keep making memes, posting links, and getting creative on the Web with
+                <a href="{{ attribution }}">respectful attribution</a>.
+              {% endtrans %}
+              </p>
+            </li>
+          </ul>
+        </div>
+
+        <div class="topic-action-moz">
+          <h3 class="topic-action-title">{{ _('How Mozilla is engaged in copyright reform') }}</h3>
+          <ul>
+            <li>
+              <h4>{{ _('Standing up for reform') }}</h4>
+              <p>
+              {% trans europe='https://blog.mozilla.org/netpolicy/2015/05/28/copyright-reform-in-the-european-union/' %}
+                Making the case for copyright reform in <a href="{{ europe }}">Europe</a>.
+              {% endtrans %}
+              </p>
+            </li>
+            <li>
+              <h4>{{ _('Advocating for protections') }}</h4>
+              <p>
+              {% trans internet_users='https://blog.mozilla.org/netpolicy/2016/03/07/challenges-to-openness-under-u-s-copyright-law/',
+                       creators='https://blog.mozilla.org/netpolicy/2017/02/21/we-are-all-creators-now/',
+                       intermediaries='https://blog.mozilla.org/netpolicy/2016/04/05/reining-in-abuses-of-the-dmca-notice-system/'
+              %}
+                Advocating for clear protections for <a href="{{ internet_users }}">internet users</a>,
```

Comment:
Copy is using Internet so far, I don't think the switch to "internet" is wanted.